### PR TITLE
Parking element

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ericlp — APPROVED** [review]

**Review by ericlp — DISMISSED** [review]

**Review by ejohnsson6 — DISMISSED** [review]

File: app/parkit/types/ParkingSpots.tsx [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-
 export interface IPosition {
-    longitude: Number;
-    latitude: Number;
+    longitude: number;
+    latitude: number;
 }
 
 export interface IParkingSpot {
     id: Number;
```
Comment (ViddeM) [inline app/parkit/types/ParkingSpots.tsx]: Should be ```number``` instead of ```Number``` right?

**Review by ViddeM — CHANGES_REQUESTED** [review]
Please fix my comment, rest is good!

File: app/parkit/types/ParkingSpots.tsx (hunk 1 above)
Comment (ghost) [reply app/parkit/types/ParkingSpots.tsx]: Done

**Review by ViddeM — APPROVED** [review]
:+1:

**Review by ejohnsson6 — APPROVED** [review]

**Review by ericlp — APPROVED** [review]

**Pull request merged**